articles/howtos/make-template-decorator-ready.md

```diff
@@ -0,0 +1,120 @@
+---
+uid: MakeTemplateDecoratorReady
+---
+# How do I make a Template, Decorator Ready?
```

Comment:
```suggestion
# How do I make a Template able to support extensibility through Decorators?
```

We may to want to change the UID and file name as well.

Reply:
Ugh, you're right but man, will need a bit of refactoring